.ci/pitest.sh

```diff
@@ -150,6 +150,7 @@ pitest-javadoc)
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        beginJavadocTree(root);</span></pre></td></tr>"
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        finishJavadocTree(root);</span></pre></td></tr>"
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        javadocTokens.clear();</span></pre></td></tr>"
+  "JavadocParamOrderCheck.java.html:<td class='covered'><pre><span  class='survived'>     if (typeDef.getType() == type) {</span></pre></td></tr>"
```

Comment:
good practice is not to extend this exceptions list. Why it is added and cannot be solved with test/removal?